0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    x = jnp.linspace(-np.pi,np.pi,100)
</pre><p>Very small things, might be better to change the "np.pi"s to "jnp.pi"s. Same with future uses. Just so that we show that it's best to use jnp instead of np as far as possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Will do, thanks! </p>